### DISCUSSION TITLE
AUR Package

### DISCUSSION THREAD

**Discussion by bephinix** [miniscruff/changie#157] [Show and tell]
As I am using **changie** on some ArchLinux/Manajaro machines, I have create a (really small) AUR package.
@miniscruff You might want to include it in your installation docs. :wink: 

Ref.: https://aur.archlinux.org/packages/changie/

**Comment by miniscruff** [comment]
I don't use those OS's can you share the full command or commands to install it?

**Reply by bephinix** [reply]
There are many ways to install AUR packages, but you can use e.g. `trizen -S changie` to perform an installation.

**Reply by miniscruff** [reply]
If you want to create a PR for it you can, it might take me a few days to get to.
